server/api/stats_test.go

```diff
@@ -197,12 +197,45 @@ func (suite *statsTestSuite) TestRegionStats() {
 			re.NoError(err)
 			stats := &statistics.RegionStats{}
 			err = apiutil.ReadJSON(res.Body, stats)
-			res.Body.Close()
+			re.NoError(res.Body.Close())
 			re.NoError(err)
 			re.Equal(data.expect.Count, stats.Count)
 			if query != "count" {
 				re.Equal(data.expect, stats)
 			}
 		}
 	}
+
+	stats11 := &statistics.RegionStats{
+		Count:                4,
+		EmptyCount:           1,
+		StorageSize:          351,
+		UserStorageSize:      291,
+		StorageKeys:          221,
+		StoreLeaderCount:     map[uint64]int{1: 1},
+		StorePeerCount:       map[uint64]int{1: 3},
+		StoreLeaderSize:      map[uint64]int64{1: 100},
+		StoreLeaderKeys:      map[uint64]int64{1: 50},
+		StorePeerSize:        map[uint64]int64{1: 301},
+		StorePeerKeys:        map[uint64]int64{1: 201},
+		StoreWriteKeys:       map[uint64]uint64{1: 0},
+		StoreWriteBytes:      map[uint64]uint64{1: 0},
+		StoreWriteQuery:      map[uint64]uint64{1: 0},
+		StoreLeaderReadBytes: map[uint64]uint64{1: 0},
+		StoreLeaderReadKeys:  map[uint64]uint64{1: 0},
+		StoreLeaderReadQuery: map[uint64]uint64{1: 0},
+		StorePeerReadBytes:   map[uint64]uint64{1: 0},
+		StorePeerReadKeys:    map[uint64]uint64{1: 0},
+		StorePeerReadQuery:   map[uint64]uint64{1: 0},
+		StoreEngine:          map[uint64]string{1: ""},
```

Comment:
Can you add some data for these?

Reply:
OK， we will add some hot stat for this test.

Reply:
done